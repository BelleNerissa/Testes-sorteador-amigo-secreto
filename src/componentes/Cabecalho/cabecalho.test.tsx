import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Cabecalho from "./index";

const mockNavegacao = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegacao
    }
})

describe('o componente de cabeÃ§alho', () => {
    test('deve ser renderizada corretamente', () => {
        const { container } = render(<RecoilRoot>
            <Cabecalho />
        </RecoilRoot>)

        expect(container).toMatchSnapshot()
    })
})