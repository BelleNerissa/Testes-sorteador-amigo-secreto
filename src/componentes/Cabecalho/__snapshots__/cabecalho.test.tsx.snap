// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`o componente de cabeçalho deve ser renderizada corretamente 1`] = `
<div>
  <header
    class="cabecalho"
  >
    <div
      aria-label="Logo do Sorteador"
      class="imagem-logo"
      role="img"
    />
    <img
      alt="Participante com um presente na mão"
      class="participante"
      src="/imagens/participante.png"
    />
  </header>
</div>
`;
